### PR TITLE
mate-panel-applet: Boilerplate

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

**Review by lukefromdc — APPROVED** [review]
This builds and runs fine. Didn't get any surprises restarting the panel with this installed, but I didn't have time to go over all the code changes to mate-panel-applet.co. There are a lot of changes with this, so needs more testing.

Neater looking code formatting than before but a few misaligned indents noticed in mate-panel-applet.h

**Comment by raveit65** [comment]
What is the meaning of `Boilerplate`?

**Comment by rbuj** [comment]
It writes a code snippet using macros:
- https://en.wikipedia.org/wiki/Boilerplate_code
- https://developer.gnome.org/gobject/stable/howto-gobject-code.html

**Comment by raveit65** [comment]
Same here, can you split out code-formatting changes from this commit please.
This would help me to understand boilerplate changes better.

**Comment by rbuj** [comment]
@raveit65 done - revert format code

File: libmate-panel-applet/mate-panel-applet.c [hunk 1]
```diff
@@ -214,18 +227,18 @@ mate_panel_applet_set_flags (MatePanelApplet      *applet,
...
+		                               "PropertiesChanged",
 					       g_variant_new ("(sa{sv}as)",
-							      MATE_PANEL_APPLET_INTERFACE,
-							      &builder,
-							      &invalidated_builder),
-					       &error);
+		                                              MATE_PANEL_APPLET_INTERFACE,
+		                                              &builder,
+		                                              &invalidated_builder),
+		                               &error);
```
Comment (raveit65) [inline libmate-panel-applet/mate-panel-applet.c:241]: It is OK to fix the alignment here if you already touch this code :)

Comment (rbuj) [reply libmate-panel-applet/mate-panel-applet.c:241]: done

**Review by raveit65 — COMMENTED** [review]
..

File: libmate-panel-applet/mate-panel-applet.c [hunk 2]
```diff
@@ -1431,15 +1512,18 @@ static void
...
+	if (priv->background == background)
 		return;
 
-	if (g_strcmp0 (applet->priv->background, background) == 0)
+	if (g_strcmp0 (priv->background, background) == 0)
 		return;
 
-	if (applet->priv->background)
-		g_free (applet->priv->background);
-	applet->priv->background = background ? g_strdup (background) : NULL;
```
Comment (raveit65) [inline libmate-panel-applet/mate-panel-applet.c:1442]: Is `if (priv->background)` line not missing here?

File: libmate-panel-applet/mate-panel-applet.c [hunk 3]
```diff
@@ -1820,35 +1925,38 @@ mate_panel_applet_constructor (GType                  type,
 {
 	GObject     *object;
 	MatePanelApplet *applet;
+	MatePanelAppletPrivate *priv;
 
 	object = G_OBJECT_CLASS (mate_panel_applet_parent_class)->constructor (type,
-	                                                                  n_construct_properties,
-	                                                                  construct_properties);
+                                                                               n_construct_properties,
+                                                                               construct_properties);
```
Comment (raveit65) [inline libmate-panel-applet/mate-panel-applet.c:1932]: Alignment.

**Comment by raveit65** [comment]
Thanks, now the commit is better understandable.
But size isn't really smaller.
I would  really like to see another reviewer.
@mate-desktop/core-team ^^^
Any way, i start testing now.

File: libmate-panel-applet/mate-panel-applet.c (hunk 2 above)
Comment (rbuj) [reply libmate-panel-applet/mate-panel-applet.c:1442]: There is no need to check if it is null before calling g_free:
```C
/**
 * g_free:
 * @mem: (nullable): the memory to free
 * 
 * Frees the memory pointed to by @mem.
 *
 * If @mem is %NULL it simply returns, so there is no need to check @mem
 * against %NULL before calling this function.
 */
void
g_free (gpointer mem)
{
  free (mem);
  TRACE(GLIB_MEM_FREE((void*) mem));
}
```

File: libmate-panel-applet/mate-panel-applet.c (hunk 3 above)
Comment (rbuj) [reply libmate-panel-applet/mate-panel-applet.c:1932]: These lines have not been modified, so I'm going undo the change

Comment (rbuj) [reply libmate-panel-applet/mate-panel-applet.c]: done

**Review by raveit65 — APPROVED** [review]
LGTM.
I saw no issues after several days of testing.

**Pull request merged**